x/protorev/types/utils.go

```diff
@@ -0,0 +1,46 @@
+package types
+
+import "encoding/binary"
+
+// Converts a uint64 to a []byte
+func UInt64ToBytes(number uint64) []byte {
+	bz := make([]byte, 8)
+	binary.LittleEndian.PutUint64(bz, number)
+	return bz
+}
+
+// Converts a []byte into a uint64
+func BytesToUInt64(bz []byte) uint64 {
+	return binary.LittleEndian.Uint64(bz)
+}
```

Comment:
There should already exist functions like this in `types`.  If note, we should define them there.

Reply:
correct, sdk implements `Uint64ToBigEndian` which will do the job. deprecating this now